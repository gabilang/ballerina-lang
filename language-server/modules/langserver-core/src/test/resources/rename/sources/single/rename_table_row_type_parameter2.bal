type Person record {|
    readonly int id;
    string name;
|};

function foo() {
    Person p = { id: 1, name: "foo" };
    table<Person> tHuman = table [{id: 1, name: "Jane"}];
    stream<Person> s = new;
}
