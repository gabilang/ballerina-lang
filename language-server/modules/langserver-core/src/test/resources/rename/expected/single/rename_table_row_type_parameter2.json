{
  "source": {
    "file": "rename_table_row_type_parameter2.bal"
  },
  "position": {
    "line": 7,
    "character": 16
  },
  "prepareRename": {
    "valid": true
  },
  "result": {
    "changes": {
      "rename_table_row_type_parameter2.bal": [
        {
          "range": {
            "start": {
              "line": 0,
              "character": 5
            },
            "end": {
              "line": 0,
              "character": 11
            }
          },
          "newText": "Student"
        },
        {
          "range": {
            "start": {
              "line": 6,
              "character": 4
            },
            "end": {
              "line": 6,
              "character": 10
            }
          },
          "newText": "Student"
        },
        {
          "range": {
            "start": {
              "line": 7,
              "character": 10
            },
            "end": {
              "line": 7,
              "character": 16
            }
          },
          "newText": "Student"
        },
        {
          "range": {
            "start": {
              "line": 8,
              "character": 11
            },
            "end": {
              "line": 8,
              "character": 17
            }
          },
          "newText": "Student"
        }
      ]
    }
  }
}
